### PR TITLE
use license apache 2.0

### PR DESCRIPTION
**Pull request by fmessmer**
no esternal contributors

do not merge before: **August, 4th 2017**
merge together with other `use license apache 2.0` PRs

### CODE REVIEW

**Comment by mathias-luedtke** [comment]
please add me as author and drop me as maintainer with this PR.

**Comment by fmessmer** [comment]
> please add me as author and drop me as maintainer with this PR.

@ipa-mdl are you ok with https://github.com/ipa320/cob_hand/pull/17/commits/013ca571f397c4e4948c5ceca9b34c7a1c827b38

File: cob_hand_bridge/package.xml [hunk 1]
```diff
@@ -4,9 +4,11 @@
   <version>0.6.1</version>
   <description>The cob_hand_bridge package</description>
 
-  <maintainer email="mathias.luedtke@ipa.fraunhofer.de">Mathias Lüdtke</maintainer>
+  <maintainer email="fmw@ipa.fhg.de">Florian Weisshardt</maintainer>
+  <maintainer email="nhg@ipa.fhg.de">Nadia Hammoudeh Garcia</maintainer>
```
Comment (NadiaHG) [inline cob_hand_bridge/package.xml:8]: why?? I have nothing to do with this package

Comment (fmessmer) [reply cob_hand_bridge/package.xml:8]: I added you in consistency with the meta-package...
Are you ok with leaving this for now and discuss future maintenance of the repo in conjunction with ipa320/care-o-bot#46

**Pull request merged**